src/frequenz/sdk/config/_config_managing.py

```diff
@@ -19,21 +20,48 @@
 
 
 class ConfigManagingActor(Actor):
-    """An actor that monitors a TOML configuration file for updates.
+    """An actor that monitors a TOML configuration files for updates.
 
-    When the file is updated, the new configuration is sent, as a [`dict`][], to the
-    `output` sender.
+    When the actor is started the configuration files will be read and sent to the
+    output sender. Then the actor will start monitoring the files for updates.
 
-    When the actor is started, if a configuration file already exists, then it will be
-    read and sent to the `output` sender before the actor starts monitoring the file
-    for updates. This way users can rely on the actor to do the initial configuration
-    reading too.
+    If no configuration file could be read, the actor will raise a exception.
+
+    The configuration files are read in the order of the paths, so the last path will
+    override the configuration set by the previous paths. Dict keys will be merged
+    recursively, but other objects (like lists) will be replaced by the value in the
+    last path.
+
+    Example:
+        If `config1.toml` contains:
```

Comment:
Maybe have an example which shows the recursive update of dict keys?

Reply:
updated